metricingester/aggworker.go

```diff
@@ -0,0 +1,123 @@
+package metricingester
+
+import (
+	"github.com/stripe/veneur/samplers"
+)
+
+type aggWorker struct {
+	samplers samplerEnvelope
+
+	inC    chan Metric
```

Comment:
If I navigated this code correctly, this is a fairly perf-critical bit; we recently (last year? @cory-stripe would know) spent a bunch of time making most channels take pointers to avoid copying. I'm not necessarily saying this should be a pointer channel, but I think I'd like to see a benchmark to ensure we're not regressing.

Reply:
Will write benchmark.

Reply:
@asf-stripe I wrote a benchmark and compared a pointer Metric to non-pointer metric in the aggWorker.Ingest() path, but there was no performance difference. On my machine, both clocked in at about 680ns/op.

Reply:
Yeah, it wasn't always obvious when go preferred a pointer to avoid copying. We tried it as a blanket thing and didn't see much difference in some cases.